src/dreyfus_fabric_cleanup.erl

```diff
@@ -18,13 +18,15 @@
 -include("dreyfus.hrl").
 -include_lib("mem3/include/mem3.hrl").
 -include_lib("couch/include/couch_db.hrl").
+-include_lib("couch_mrview/include/couch_mrview.hrl").
```

Comment:
What's this import for? This links dreyfus to couch_mrview which is not a good idea.